dbstore.go

```diff
@@ -612,8 +614,10 @@ func (p *persister) save(ctx context.Context, ws *Worksheet) error {
 				})
 			}
 		}
-		if _, err := insertSliceElements.ExecContext(ctx); err != nil {
-			return err
+		if atLeastOneSliceElementExists {
```

Comment:
Annoyingly the insert builder does not expose `records` which we could check the `len()` of.